sql/src/jmh/java/io/crate/operation/collect/collectors/LuceneBatchIteratorBenchmark.java

```diff
@@ -95,4 +95,45 @@ public void measureConsumeLuceneBatchIterator(Blackhole blackhole) throws Except
             blackhole.consume(input.value());
         }
     }
+
+    private class CustomIntegerColumnReference extends LuceneCollectorExpression<Integer> {
```

Comment:
Maybe add a comment that this is a variant of IntegerColumnReference that doesn't involve the fieldCache to avoid having to create it in the test and that it wouldn't be used anyway as the index contains docvalues